tests/integration-tests/tests/test_efa/test_efa/test_efa/pcluster.config.ini

```diff
@@ -0,0 +1,22 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = {{ scheduler }}
+master_instance_type = t2.micro
+compute_instance_type = {{ instance }}
+initial_queue_size = 2
```

Comment:
why do we need 2 compute nodes?

Reply:
To run an MPI job that communicates between nodes